43/README.md

```diff
@@ -235,24 +233,26 @@ ZRE uses UDP IPv4 *beacon* broadcasts to discover nodes and track their presence
 * Each ZRE node SHALL broadcast, at regular intervals, a UDP beacon that identifies itself to any listening nodes on the network.
 * When a ZRE node receives a beacon from a node that it does not already know about, it SHALL consider this to be a new peer.
 * When a ZRE node stops receiving beacons from a peer that it knows about, after a certain interval it SHALL consider this peer to be disconnected or dead.
+* A ZRE node configured with CURVE security SHOULD ignore beacons that do not contain a valid 32-octet CURVE key.
```

Comment:
that thought had crossed my mind- i sometimes default to "SHOULD" when i'm not sure- leaving it up to the last mile... but mulling it over, at-least SHALL if not MUST.

i'm OK with MUST- just wasn't sure if we had considered all scenarios before making that suggestion... (i can't think of any reason WHY, but that's why i'm soliciting feedback too).